modules/travel_pay/spec/services/claims_association_service_spec.rb

```diff
@@ -240,25 +256,26 @@
       expect(appt_with_claim['travelPayClaim']['claim']).to be_nil
     end
 
-    it 'returns appointment with error metadata if claims call fails' do
-      allow_any_instance_of(TravelPay::ClaimsService)
-        .to receive(:get_claims_by_date_range)
-        .with(
-          { 'start_date' => '2024-01-01T16:45:34Z',
-            'end_date' => '2024-01-01T16:45:34Z' }
-        )
-        .and_return(nil)
+    # TODO: needs work
+    # it 'returns appointment with error metadata if claims call fails' do
+    #   allow_any_instance_of(TravelPay::ClaimsService)
+    #     .to receive(:get_claims_by_date_range)
+    #     .with(
+    #       { 'start_date' => '2024-01-01T16:45:34Z',
+    #         'end_date' => '2024-01-01T16:45:34Z' }
+    #     )
+    #     .and_return(nil)
```

Comment:
commented leftovers? If the TODO is relevant, can we specify what work is needed?

Reply:
🤦‍♀️ Oops. I'll do the TODO and fix it up. Nice catch! 😁

Reply:
✅